src/components/icon-as-button/IconAsButton.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon, {types, colors as iconColors} from '../icons/Icon';
+
+const colors = {
```

Comment:
Can we have some const with all of it (external to components) ?
@vergilius WDYT ? we just need to verify how many components uses all available possible colors